aleth/main.cpp

```diff
@@ -828,8 +790,7 @@ int main(int argc, char** argv)
         chainParams.allowFutureBlocks = true;
 
     dev::WebThreeDirect web3(WebThreeDirect::composeClientVersion("aleth"), db::databasePath(),
-        snapshotPath, chainParams, withExisting, nodeMode == NodeMode::Full ? caps : set<string>(),
-        netPrefs, &nodesState, testingMode);
+        snapshotPath, chainParams, withExisting, caps, netPrefs, &nodesState, testingMode);
```

Comment:
`caps` parameter doesn't do anything useful anymore, we could remove it

Reply:
Confirmed - it's only used in the WebThree constructor to determine if the client should be configured / started and we always do this when running Aleth so it can be removed.